### ISSUE TITLE
How to split the training set and the validation set?

### ISSUE THREAD

**Issue by develduan** [chenyaofo/pytorch-cifar-models#1]
Thanks for your pretrained-models on CIFAR10, I want to do some test with pretrained model, howerer, the inference acc(99.624%) seems too high (using `torchvision.datasets.CFAR10`). I think your train/val split approach is different with torchvision, could you tell me how your data set is divided?

**Comment by develduan** [comment]
sorry for my  mistake, I used the wrong data loader. After the bug fixed, howerver, I can not reach the number(Resnet20@acc@1: 91.65), my inference acc is about 90.5 and I wonder what the validaton data augumentation? is it the same  as traning?
